### ISSUE TITLE
KivyMD callback() doubt

### ISSUE THREAD

**Issue by pj-mathematician** [lucianolivares/Tutorial-Kivymd#7]
Sir, can you explain me the usage of callback() in MDFloatingActionButtonSpeedDial i want to have separate functions for all the sub buttons saved in data dictionary, i could not find it anywhere, so I thought you can help me

**Comment by lucianolivares** [comment]
> Sir, can you explain me the usage of callback() in MDFloatingActionButtonSpeedDial i want to have separate functions for all the sub buttons saved in data dictionary, i could not find it anywhere, so I thought you can help me

Hello, if I understand correctly you want to call different functions on your buttons using callback, in my script I did it using if conditionals depending on the parameter I gave when creating the button to execute different functions. But I think that to use callback you must give the button the function as a parameter and then instantiate it inside, then use it. Maybe this link can help you. In the same way, I will try to make a tutorial explaining this topic in more detail.

https://code-maven.com/function-or-callback-in-python
